terraform/aws/security/aws-elb-access-logs-not-enabled.yaml

```diff
@@ -0,0 +1,40 @@
+rules:
+- id: aws-elb-access-logs-not-enabled
+  patterns:
+  - pattern-either:

```

Comment:
I think this rule is missing clauses for the `aws_alb` resource in addition to `aws_lb`

Reply:
Yes, `aws_alb` is missing and needs fixing